<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>643</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>985</width>
    <height>643</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>985</width>
    <height>643</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>    </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/dcristian/.designer/screenshotMaker/icon.png</normaloff>
    <normalon>icon.ico</normalon>C:/Users/dcristian/.designer/screenshotMaker/icon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="actionButton">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SNAPSHOT</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="popupIdField">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="maxVisibleItems">
          <number>50</number>
         </property>
         <property name="duplicatesEnabled">
          <bool>false</bool>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="images">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>965</width>
         <height>566</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>965</width>
         <height>566</height>
        </size>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab1">
        <attribute name="title">
         <string>Screenshot 1</string>
        </attribute>
        <widget class="QLabel" name="labelTab1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>960</width>
           <height>540</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>960</width>
           <height>540</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>960</width>
           <height>540</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="saveButton1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>2</y>
           <width>85</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab2">
        <attribute name="title">
         <string>Screenshot 2</string>
        </attribute>
        <widget class="QLabel" name="labelTab2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>960</width>
           <height>540</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>960</width>
           <height>540</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>960</width>
           <height>540</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="saveButton2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>2</y>
           <width>85</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
